UI/MapWnd.cpp

```diff
@@ -6960,89 +6972,89 @@ namespace {
 void MapWnd::ConnectKeyboardAcceleratorSignals() {
     HotkeyManager* hkm = HotkeyManager::GetManager();
 
-    hkm->Connect(boost::bind(&MapWnd::ReturnToMap, this), "map.return_to_map",
+    hkm->Connect(boost::bind(&MapWnd::ReturnToMap, this), "ui.map.open",
                  AndCondition({VisibleWindowCondition(this), NoModalWndsOpenCondition}));
-    hkm->Connect(boost::bind(&MapWnd::OpenMessages, this), "map.open_chat",
+    hkm->Connect(boost::bind(&MapWnd::OpenMessages, this), "ui.map.messages.open",
```

Comment:
this is named inconsistently with the rest...

Reply:
`ui.map.messages.open.hotkey` is distinct from `ui.map.messages.hotkey`. The former forces the window open, the later toggles the window open or closed.

If no objection, I will remove the former.

Reply:
Several other window toggle command exist and have hotkeys... particularly those with top-right buttons... so not having one for messages may be odd. (if that's what you're suggesting...)

Reply:
To clarify: A hotkey option to toggle messages window open/closed will still exist as `ui.map.messages.hotkey`

In current master there are two hotkeys for this window:
*  [`UI.hotkeys.map.messages `](https://github.com/freeorion/freeorion/blob/master/UI/MapWnd.cpp#L173) : Toggles window open/closed
*  [`UI.hotkeys.map.open_chat `](https://github.com/freeorion/freeorion/blob/master/UI/MapWnd.cpp#L166) : Opens window

I've removed the later (and `MapWnd::OpenMessages`, as it is now unused)

Reply:
ok